faktory/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if and (.Values.metrics.enabled) (.Values.metrics.serviceMonitor.enabled) }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "faktory.fullname" . }}
+  {{- if .Values.metrics.serviceMonitor.namespace }}
+  namespace: {{ .Values.metrics.serviceMonitor.namespace }}
+  {{- else }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+  labels:
+    app: {{ template "faktory.name" . }}
```

Comment:
How do you feel about using the longer-form label keys for consistency?
e.g.
`app.kubernetes.io/instance: ...`, etc

Reply:
Done